surf/apps/locale/models.py

```diff
@@ -4,17 +4,35 @@
 
 
 class Locale(UUIDModel):
-    asset = models.CharField('Asset ID', max_length=512, unique=True)
-
-    en = models.CharField('English, en', max_length=5120,
-                          null=True, blank=True)
 
-    nl = models.CharField('Dutch, nl-NL',
-                          max_length=5120, null=True, blank=True)
+    asset = models.CharField('Asset ID', max_length=512, unique=True)
+    en = models.CharField('English, en', max_length=5120, null=True, blank=True)
+    nl = models.CharField('Dutch, nl', max_length=5120, null=False, blank=False)
+    is_fuzzy = models.BooleanField(default=False)
```

Comment:
We're adding the is_fuzzy column now, because we need it for upcoming stories. That saves us another database migration on production.